Duplicati/Library/Backend/Telegram/ChannelFileInfo.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using Duplicati.Library.Common.IO;
+using TeleSharp.TL;
+
+namespace Duplicati.Library.Backend
+{
+    public class ChannelFileInfo : IEquatable<ChannelFileInfo>
+    {
+        public int MessageId { get; set; }
+        public long MediaDocAccessHash { get; set; }
+        public long DocumentId { get; set; }
+        public int Version { get; set; }
+        public long Size { get; set; }
+        public string Name { get; set; }
+        public DateTime Date { get; set; }
+        public long AccessHash { get; set; }
```

Comment:
The `AccessHash` property appears to be unused.  Can we remove it?

Reply:
Surely